course/page/choice.py

```diff
@@ -387,7 +404,7 @@ def normalized_bytes_answer(self, page_context, page_data, answer_data):
         permutation = page_data["permutation"]
 
         if answer_data is None:
-            unpermuted_choice = None
+            return None
```

Comment:
I modified here so that page submitted with no answer won't generate a json file.